poc_mobile_verifier/src/speedtests.rs

```diff
@@ -93,7 +97,9 @@ impl SpeedtestRollingAverage {
         // Write out the speedtests to S3
         let average = Average::from(&self.speedtests);
         let validity = average.validity() as i32;
-        let reward_multiplier = average.reward_multiplier();
+        // this is guaratneed to safely convert and not panic as it can only be one of
+        // four possible decimal values based on the speedtest average tier
+        let reward_multiplier = average.reward_multiplier().try_into().unwrap();
```

Comment:
if guaranteed to not panic, why then use try_info and not a plain old From/Into impl ?

Reply:
Decimal only implements TryFrom/Into for floats